samples/snippets/create/create_with_container_no_mounting.py

```diff
@@ -0,0 +1,63 @@
+#  Copyright 2022 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# [START batch_create_container_job]
+from google.cloud import batch_v1
+
+
+def create_container_job(project_id: str, region: str, job_name: str) -> batch_v1.Job:
+    client = batch_v1.BatchServiceClient()
+
+    group = batch_v1.TaskGroup()
+
+    task = batch_v1.TaskSpec()
+    runnable = batch_v1.Runnable()
+    runnable.container = batch_v1.Runnable.Container()
+    runnable.container.image_uri = "gcr.io/google-containers/busybox"
+    runnable.container.entrypoint = "/bin/sh"
+    runnable.container.commands = ["-c", "echo Hello world! This is task ${BATCH_TASK_INDEX}. This job has a total of ${BATCH_TASK_COUNT} tasks."]
+    task.runnables = [runnable]
+
+    resources = batch_v1.ComputeResource()
+    resources.cpu_milli = 2000
+    resources.memory_mib = 16
+    task.compute_resource = resources
+
+    task.max_retry_count = 2
+    task.max_run_duration = "3600s"
+
+    group.task_count = 4
+    group.task_spec = task
+
+    allocation_policy = batch_v1.AllocationPolicy()
+    policy = batch_v1.AllocationPolicy.InstancePolicy()
+    policy.machine_type = "e2-standard-4"
+    instances = batch_v1.AllocationPolicy.InstancePolicyOrTemplate()
+    instances.policy = policy
+    allocation_policy.instances = [instances]
+
+    job = batch_v1.Job()
```

Comment:
I see the sample is split into "paragraphs" - a bunch of related lines separated by whitespace. It would be easier to navigate the samples if each "paragraph" had a "heading" - a comment summarizing what's happening in it on a high level.

Reply:
I added comments and reorganized the lines a bit.